src/ops/ops.go

```diff
@@ -510,7 +510,12 @@ func (o *ops) getRaidDevices2Members() (map[string][]string, error) {
 		}
 
 		fields := strings.Fields(lines[i])
-		raidDeviceName := fields[1]
+		// In case of symlink, get real file path
+		raidDeviceName, err := filepath.EvalSymlinks(fields[1])
+		if err != nil {
+			return nil, err
```

Comment:
There should be much more of this in the file but let's at-least make sure new code has it - errors should be given context, otherwise it's hard to follow where they come from

```suggestion
			return nil, errors.Wrapf(err, "failed to evaluate raid symlink")
```

Reply:
sure!
Just added your suggestion :)